tests/integration/test_endpoints.py

```diff
@@ -31,6 +31,14 @@ def test_query(self):
         self.assertEqual(response.json['query'], query)
         self.assertGreaterEqual(response.json['results'][0]['count'], 0)
 
+        query = "SELECT count(*) FROM FILES WHERE SIZE > :s;"
```

Comment:
would it be worth it to add a test showing this gives the same result as running the query without params 
`"SELECT count(*) FROM FILES WHERE SIZE > 0;"`
Also would it be worth it to return the query that was actually run with the params in place to help debug issues?

Reply:
1. Yes, but if the database undergoes a change in the number of files as we run the test, the result will change. So we need some way to ensure atomicity in the test. I tried passing multiple queries (`select count(*) from files; select count(*) from files where size > 0;` but the first result gets shadowed by the second. Do you know how to ensure atomicity?

2. Yes, that's a great idea but I'd like to tackle it separately. Filed #186 to track that.

Reply:
Maybe if you reset the db connection between queries?